sde-cookbook/dlt-viewer/install_dlt-viewer.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) Luxoft 2018
+#
+# All rights reserved
+#
+
+DLT_VIEWER_TMP_DIR=/tmp/github
+
+DLT_REPO_NAME="dlt-viewer"
+
+mkdir -p ${DLT_VIEWER_TMP_DIR}
+cd ${DLT_VIEWER_TMP_DIR}
+git clone https://github.com/GENIVI/${DLT_REPO_NAME}
+cd ${DLT_REPO_NAME}
+mkdir build
+cd build
+if ! type "qmake" > /dev/null; then
+	echo "Missing qmake-qt5 installation : dlt-viewer not installed"
+	exit 1
+fi
+qmake ../BuildDltViewer.pro
+make
```

Comment:
I'd like to make sure we're building multi-threaded to speed builds up. See e.g. https://github.com/Pelagicore/gdp-qtas-sde/blob/master/Vagrantfile how we used makeflags to speed builds up.